website/versioned_docs/version-5.x/ecosystem-css.md

```diff
@@ -0,0 +1,336 @@
+
+SASS and other build tools often produce global CSS rather than [scoped css](#scoped-css), which can be undesireable in a microfrontends architecture because it can result in css class name collisions between your microfrontends. To avoid this, you can use [SASS modules](https://blog.bitsrc.io/how-to-use-sass-and-css-modules-with-create-react-app-83fa8b805e5e) (or similar) to scope the CSS.
+
+## Webpack CSS resources
+
+Below is a list of commonly used webpack loaders and plugins that can help with loading CSS:
+
+- [css-loader](https://webpack.js.org/loaders/css-loader/#root) facilitates using CSS Modules and properly handling `@import()` within CSS files.
+- [style-loader](https://webpack.js.org/loaders/style-loader/#root) facilitates mounting CSS via `<style>` elements. This is often used in development mode, but not production.
+- [postcss-loader](https://webpack.js.org/loaders/postcss-loader/#root) is similar to CSS modules, but for more advanced use cases that require PostCSS.
+- [sass-loader](https://webpack.js.org/loaders/sass-loader/#root) can be used to compile SASS to CSS.
+- [single-spa-css](#single-spa-css) can be used to automatically detect which CSS files to load during the `mount` lifecycle function of your single-spa application or parcel.
+
+## single-spa-css
+
+The `single-spa-css` npm package implements helper functions for loading, mounting, and unmounting css. It does this by adding `<link rel="stylesheet">` elements to the DOM to mount the CSS, and removing the `<link>` from the DOM when it's time to unmount the CSS.
+
+### Installation
+
+```sh
+npm install single-spa-css
+
+pnpm install single-spa-css
+
+yarn add single-spa-css
+```
+
+### Usage
+
+```js
+import singleSpaCss from 'single-spa-css';
+
+const cssLifecycles = singleSpaCss({
+  // required: a list of CSS URLs to load
+  // can be omitted if webpackExtractedCss is set to true, do not specify webpack extracted css files here
+  cssUrls: ['https://example.com/main.css'],
+
+  // optional: defaults to false. This controls whether extracted CSS files from webpack
+  // will automatically be loaded. This requires using the ExposeRuntimeCssAssetsPlugin,
+  // which is documented below.
+  webpackExtractedCss: false,
+
+  // optional: defaults to true. Indicates whether the <link> element for the CSS will be
+  // unmounted when the single-spa microfrontend is unmounted.
+  shouldUnmount: true,
+
+  // optional: defaults to 5000. The number of milliseconds to wait on the <link> to load
+  // before failing the mount lifecycle.
+  timeout: 5000
+})
+
+const reactLifecycles = singleSpaReact({...})
+
+// Export an array of lifecycles to integrate with a framework's
+// single-spa lifecycles. The order matters.
+export const bootstrap = [
+  cssLifecycles.bootstrap,
+  reactLifecycles.bootstrap
+]
+
+export const mount = [
+  // The css lifecycles should be before your framework's mount lifecycle,
+  // to avoid a Flicker of Unstyled Content (FOUC)
```

Comment:
```suggestion
  // to avoid a Flash of Unstyled Content (FOUC)
```

I've always read/seen it as Flash?

Reply:
Ah okay I probably just made up Flicker at some point because I forgot